freq/lizard_connector.py

```diff
@@ -771,6 +768,24 @@ def from_timeseries_uuid(self, uuid):
         return {}
 
 
+class Users(Base):
+    data_type = "users"
+
+    def get_organisations(self, username):
+        self.get(username=username)
+        if len(self.results) > 1 or len(self.results) == 0:
+            if len(self.results):
+                print(self.results)
+                raise LizardApiError("Username is not unique")
+            raise LizardApiError("Username not found")
+        organisations_url = self.results[0].get("organisations_url")
```

Comment:
Ik zie hier niet wat die `organisations_url` precies is en dus welke "query" je ermee doet, maar als het werkt zal het wel goed zijn.